### ISSUE TITLE
IndexOutOfBoundsException when reading getColumnDataAsMap()

### ISSUE THREAD

**Issue by markleusink** [klehmann/domino-jna#17]
Seems to happen if the view that's being read doesn't have any sorting. Stacktrace:

```
04-01-2018 20:29:07   HTTP JVM: java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
04-01-2018 20:29:07   HTTP JVM: 	at java.util.ArrayList.rangeCheck(ArrayList.java:664)
04-01-2018 20:29:07   HTTP JVM: 	at java.util.ArrayList.get(ArrayList.java:440)
04-01-2018 20:29:07   HTTP JVM: 	at com.mindoo.domino.jna.NotesCollection.scanColumnsNew(NotesCollection.java:2884)
04-01-2018 20:29:07   HTTP JVM: 	at com.mindoo.domino.jna.NotesCollection.scanColumns(NotesCollection.java:2847)
04-01-2018 20:29:07   HTTP JVM: 	at com.mindoo.domino.jna.NotesCollection.getColumnNames(NotesCollection.java:2801)
04-01-2018 20:29:07   HTTP JVM: 	at com.mindoo.domino.jna.NotesViewEntryData.getColumnNames(NotesViewEntryData.java:510)
04-01-2018 20:29:07   HTTP JVM: 	at com.mindoo.domino.jna.NotesViewEntryData.getColumnDataAsMap(NotesViewEntryData.java:538)
```

(I'm using the latest release: 0.9.10)

**Comment by klehmann** [comment]
Commit 23e525602a6ff0282034a5b227cc0a8e1712f918 closes #17

**Comment by klehmann** [comment]
Thanks for reporting!
